src/components/NavBar/MainMenu.tsx

```diff
@@ -1,54 +1,80 @@
 import React from 'react';
-import { Colors } from '../../constants';
+import { Colors, Styles } from '../../constants';
 import { SbNavLink } from '../SbNavLink';
 import { Home, Search, Eye, Grid } from 'react-feather';
+import { Link } from 'react-router-dom';
 
 export interface MenuItemProps {
   name: string;
-  icon: JSX.Element;
 }
 
 export interface IconProps {
   size: number;
   color: string;
 }
 
-export const MenuItem = ({ icon, name }: MenuItemProps): JSX.Element => (
+export const IconStyle: IconProps = {
```

Comment:
This and `MobileIconStyle` should be camelCased, since they're plain JSON objects.

Reply:
You might also consider putting these in the `constants` file, since that's what you're treating them as.